src/main/java/com/revature/aspects/LoggingAspect.java

```diff
@@ -40,16 +42,21 @@ public void setRequest(HttpServletRequest request, HttpServletResponse response)
 	 * 2) request bodies except for sensitive locations (Login)
 	 * 3) exceptions thrown by a controller
 	 */
+	@SuppressWarnings("rawtypes")
 	@Around("within(com.revature.controllers..*)")
 	public Object logControllers(ProceedingJoinPoint jp) throws Throwable {
```

Comment:
What are the changes here for?

Reply:
The array list below can be of any type as it is an array of unknown objects being returned to the user in the response. As such I cannot use an ArrayList<User> or ArrayList<Driver>, etc. as it might be any of them... and the function I'm writing just uses a .toString so it really does not care what is inside the array.